makefiles/sudo.mk

```
@@ -42,8 +42,11 @@ endif
 		--with-password-timeout=0 \
 		--with-passprompt="[sudo] password for %p: " \
 		sudo_cv___func__=yes \
+		ac_cv_have_working_snprintf=yes \
+		ac_cv_have_working_vsnprintf=yes \
 		$(SUDO_CONFIGURE_ARGS)
-	+$(MAKE) -C $(BUILD_WORK)/sudo
+	+$(MAKE) -C $(BUILD_WORK)/sudo \
+		SUDO_LDFLAGS='$(LDFLAGS) -static' # sudo passes a flag to libtool twice, one of which is passed to the linker
```

Do it like this instead.

```suggestion
		SUDO_LDFLAGS="$(LDFLAGS) -static" # sudo passes a flag to libtool twice, one of which is passed to the linker
```